content/departments/sales/tools/referral_and_reseller_policy.md

```diff
@@ -0,0 +1,85 @@
+**<span style="text-decoration:underline;">Referral Program Summary</span>**
+
+- <span style="text-decoration:underline;">Program Purpose</span>: The Sourcegraph Business Development team wants to enlist the partner community to accelerate Sourcegraph’s growth. The primary impetus of referrals should be that Sourcegraph provides meaningful business benefits to the end customers. Referral payments are important part of certain partners’ business models and Sourcegraph will enthusiastically pay them for accelerating our customer acquisition.
+- <span style="text-decoration:underline;">Partner Nomination:</span> Potential referral partners should be referred to the Head of Business Development
+- <span style="text-decoration:underline;">Agreement</span>: The partner referral agreement is available [here](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.
```

Comment:
Links need to have a descriptive name for accessibility. Instead of using the word "here", rephrase the sentence and link the part of the sentence that describes the document, e.g.

> **Agreement**: Use our [partner referral agreement template](https://docs.google.com/document/d/1UnEJ2Lzw0slRkquHWjjpsO0KN7SVfZgy/edit). Note that this will eventually be architected to stand as an addendum to a standard partnership agreement.